README.md

```diff
@@ -30,14 +30,14 @@
    * **Language detection**: Artemis detects the language of the submission and shows the word and character count. 
 5. **[File upload exercises](https://docs.artemis.cit.tum.de/user/exercises/file-upload/)** with manual assessment
 6. **[Exam mode](https://docs.artemis.cit.tum.de/user/exam_mode/)**: Instructors can create online exams with exercise variants, integrated plagiarism checks, test runs and student reviews. You can find more information on [Exam mode student features](https://artemis.cit.tum.de/features/students) and on [Exam mode instructor features](https://artemis.cit.tum.de/features/instructors).
-7. **[Grading](https://docs.artemis.cit.tum.de/user/grading/)**: Instructors can configure grade keys for courses and exams to automatically calculate grades and display them to students. Grades can be easily exported as csv files to upload them into university systems (such as Campus online)
+7. **[Grading](https://docs.artemis.cit.tum.de/user/grading/)**: Instructors can configure grade keys for courses and exams to automatically calculate grades and display them to students. Grades can be easily exported as csv files to upload them into university systems (such as Campus online). Instructors can optionally define bonus configurations for final exams to improve student grades according to their grades from a midterm exam or course exercises.
```

Comment:
Should this be mentioned here or in the Exam section?

Reply:
Good question, I thought about it a bit more and keeping it here seemed better for me because:

- While the bonus is applied to the exam, the bonus source can also be a course so this feature is related to both courses and exams.
- Bonus feature cannot be used without defining a grading key first and it is closely related to the grading steps.
- (Not a great argument but can be a tiebreaker) Exam documentation already contains lots of topics so this could at least take some of the heavy load there to the grading documentation which is shorter.